[2023-03-15 08:45:03,968]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 08:45:04,877]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 10:07:03,804]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 10:16:56,218]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 10:16:57,530]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 10:19:57,710]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 10:19:58,356]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 10:21:35,110]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 10:21:35,792]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 10:23:36,264]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 10:23:36,988]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 10:27:52,444]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 10:30:21,682]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 10:58:50,895]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 10:58:51,754]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 11:02:23,034]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 11:07:18,169]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 11:14:22,019]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 12:24:21,380]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 12:32:05,755]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 12:32:06,220]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 12:32:47,395]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 12:32:47,891]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:02:41,408]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:02:41,990]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:04:29,407]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:04:29,894]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:06:04,683]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:06:05,210]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:07:39,888]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:07:40,405]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:10:55,087]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:13:21,723]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:13:25,577]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:14:22,746]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:15:26,438]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:15:28,326]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:16:59,424]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:16:59,976]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:19:21,088]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:19:21,603]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:21:10,836]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:21:11,362]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:26:01,547]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:26:02,087]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:26:37,329]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:26:37,866]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:28:13,646]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:28:14,155]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:33:07,362]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:33:07,876]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:36:34,353]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:36:34,863]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:37:58,097]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:37:58,670]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:40:58,693]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:41:00,209]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:43:34,339]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:43:34,910]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:47:34,462]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 13:47:35,000]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 13:51:25,893]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:05:55,434]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:08:23,011]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:15:59,771]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:17:32,357]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:17:32,886]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:28:08,174]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:28:08,713]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:31:46,051]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:31:46,560]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:31:54,982]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:32:54,680]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:34:40,226]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:34:40,724]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:36:31,915]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:36:32,398]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:36:56,483]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:38:42,409]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:39:32,146]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:41:44,913]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:44:18,082]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:46:22,929]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:49:20,167]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 14:50:23,037]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 14:54:41,923]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 15:00:11,938]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 15:05:49,694]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\tornado服务器\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 18:48:48,299]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 19:13:30,502]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 19:13:30,929]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 19:27:12,805]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 19:58:00,954]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 20:03:58,784]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 20:05:50,732]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 19, in <module>
    server.listen(server_port, address=server_ip)
  File "D:\Anaconda\lib\site-packages\tornado\tcpserver.py", line 151, in listen
    sockets = bind_sockets(port, address=address)
  File "D:\Anaconda\lib\site-packages\tornado\netutil.py", line 161, in bind_sockets
    sock.bind(sockaddr)
OSError: [WinError 10049] 在其上下文中，该请求的地址无效。

[2023-03-15 20:09:48,892]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 19, in <module>
    server.listen(server_port, address=server_ip)
  File "D:\Anaconda\lib\site-packages\tornado\tcpserver.py", line 151, in listen
    sockets = bind_sockets(port, address=address)
  File "D:\Anaconda\lib\site-packages\tornado\netutil.py", line 161, in bind_sockets
    sock.bind(sockaddr)
OSError: [WinError 10049] 在其上下文中，该请求的地址无效。

[2023-03-15 20:10:51,991]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 20:19:10,504]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 20:19:12,373]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 20:21:22,737]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 20:23:16,570]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 19, in <module>
    server.listen(server_port, address=server_ip)
  File "D:\Anaconda\lib\site-packages\tornado\tcpserver.py", line 151, in listen
    sockets = bind_sockets(port, address=address)
  File "D:\Anaconda\lib\site-packages\tornado\netutil.py", line 161, in bind_sockets
    sock.bind(sockaddr)
OSError: [WinError 10049] 在其上下文中，该请求的地址无效。

[2023-03-15 20:25:23,703]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 19, in <module>
    server.listen(server_port, address=server_ip)
  File "D:\Anaconda\lib\site-packages\tornado\tcpserver.py", line 151, in listen
    sockets = bind_sockets(port, address=address)
  File "D:\Anaconda\lib\site-packages\tornado\netutil.py", line 161, in bind_sockets
    sock.bind(sockaddr)
OSError: [WinError 10049] 在其上下文中，该请求的地址无效。

[2023-03-15 20:27:48,880]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 19, in <module>
    server.listen(server_port, address=server_ip)
  File "D:\Anaconda\lib\site-packages\tornado\tcpserver.py", line 151, in listen
    sockets = bind_sockets(port, address=address)
  File "D:\Anaconda\lib\site-packages\tornado\netutil.py", line 161, in bind_sockets
    sock.bind(sockaddr)
OSError: [WinError 10049] 在其上下文中，该请求的地址无效。

[2023-03-15 20:28:11,657]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 20:40:38,682]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 20:40:39,248]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 20:45:40,128]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 21:09:53,219]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 21:11:41,316]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 21:11:41,735]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 21:27:00,122]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

[2023-03-15 21:27:01,710]-[root]-[INFO]-[main_server.py:<module>():20 Line]:AiServer启动成功!
[2023-03-15 21:30:49,314]-[root]-[ERROR]-[main_server.py:<module>():24 Line]:Traceback (most recent call last):
  File "D:\上体蓝本\tornado_template\main_server.py", line 22, in <module>
    tornado.ioloop.IOLoop.instance().start()
  File "D:\Anaconda\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 316, in run_forever
    super().run_forever()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 596, in run_forever
    self._run_once()
  File "D:\Anaconda\lib\asyncio\base_events.py", line 1854, in _run_once
    event_list = self._selector.select(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 434, in select
    self._poll(timeout)
  File "D:\Anaconda\lib\asyncio\windows_events.py", line 783, in _poll
    status = _overlapped.GetQueuedCompletionStatus(self._iocp, ms)
KeyboardInterrupt

